src/sinol_make/commands/run/__init__.py

```diff
@@ -79,6 +79,7 @@ def print_view(term_width, term_height, program_groups_scores, all_results, prin
     title = 'Done %4d/%4d. Time remaining (in the worst case): %5d seconds.' \
             % (print_data.i + 1, len(executions), time_remaining)
     title = title.center(term_width)
+    NO_STATUS = "  "
```

Comment:
Then why not also have similar "enums" for other status?

Reply:
I added an enum for statuses